packages/gatsby/src/commands/build-html.js

```diff
@@ -13,7 +13,7 @@ module.exports = async (program: any) => {
 
   debug(`generating static HTML`)
   // Reduce pages objects to an array of paths.
-  const pages = store.getState().pages.map(page => page.path)
+  const pages = [...store.getState().pages.values()].map(page => page.path)
```

Comment:
`Array.from(store.getState().pages.values(), page => page.path)` maybe?

Reply:
Oh haha — didn't know this was part of `Array.from`. Yeah, totally makes sense.

Reply:
i just learned it myself!